substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/TimeZoneSubstitutions.java

```diff
@@ -78,12 +78,23 @@ public static TimeZone getTimeZone(String id) {
     static class Options {
         @Option(help = "When true, all time zones will be pre-initialized in the image.")//
         public static final HostedOptionKey<Boolean> IncludeAllTimeZones = new HostedOptionKey<>(false);
+
+        @Option(help = "A comma separated list of time zones that will be pre-initialized in the image.")//
+        public static final HostedOptionKey<String> IncludeTimeZones = new HostedOptionKey<>(null);
     }
 
     @Override
     public void afterRegistration(AfterRegistrationAccess access) {
         TimeZone defaultZone = TimeZone.getDefault();
-        String[] supportedZoneIDs = Options.IncludeAllTimeZones.getValue() ? TimeZone.getAvailableIDs() : new String[]{"GMT", "UTC", defaultZone.getID()};
+        final String[] supportedZoneIDs;
+        if (Options.IncludeAllTimeZones.getValue()) {
+            supportedZoneIDs = TimeZone.getAvailableIDs();
+        } else if (Options.IncludeTimeZones.hasBeenSet()) {
+            final String[] timeZones = Options.IncludeTimeZones.getValue().split(",");
+            supportedZoneIDs = Arrays.stream(timeZones).map(String::trim).toArray(String[]::new);
+        } else {
+            supportedZoneIDs = new String[]{"GMT", "UTC", defaultZone.getID()};
```

Comment:
This line feels wrong. Would it be possible to make this a default value of `IncludeTimeZones`?

Reply:
Hello @vjovanov,

Thank you very much for your review and sorry about the late response.

I've taken your input on this one and updated this PR to include this change. These values now form the default value for the new option.